src/components/grade/AllSelect.tsx

```diff
@@ -0,0 +1,66 @@
+import { IElement } from '@/pages/GradeProgramPage';
+import { Text, theme } from '@team-entry/design_system';
+import React, { SetStateAction, useEffect, useState } from 'react';
+import styled from '@emotion/styled';
+
+export type Grade = 'A' | 'B' | 'C' | 'D' | 'E' | 'X';
+
+interface PropsType {
+  current: number;
+  element?: IElement[][];
+  setElement?: React.Dispatch<SetStateAction<IElement[][]>>;
+}
+
+const AllSelect = ({ current, element, setElement }: PropsType) => {
+  const [grade, setGrade] = useState<Grade>('A');
+  const arr = ['A', 'B', 'C', 'D', 'E'];
```

Comment:
```suggestion
  const arr:Grade[] = ['A', 'B', 'C', 'D', 'E'];
```
로 바꿔고 컴포넌트 상단으로 빼내주세요.
상수라서 빼놓는게 좋을거 같아요